control/controlknobs/controlknobs.go

```diff
@@ -64,6 +64,10 @@ type Knobs struct {
 	// LinuxForceNfTables is whether the node should use nftables for Linux
 	// netfiltering, unless overridden by the user.
 	LinuxForceNfTables atomic.Bool
+
+	// SeamlessKeyRenewal is whether to enable the beta functionality of
+	// renewing node keys without breaking connections.
+	SeamlessKeyRenewal atomic.Bool
```

Comment:
should this be double opt-in?

Reply:
Discussed on Slack & noted in the Design Document (but TLDR: no, double opt-in is not necessary for this)